docs/en/tutorials/getting-started.rst

```diff
@@ -31,9 +31,8 @@ The code of this tutorial is `available on Github <https://github.com/doctrine/d
 What is Doctrine?
 -----------------
 
-Doctrine 2 is an `object-relational mapper (ORM)
-<http://en.wikipedia.org/wiki/Object-relational_mapping>`_ for PHP 5.4+ that
-provides transparent persistence for PHP objects. It uses the Data Mapper
+Doctrine 2 is an `object-relational mapper (ORM) <http://en.wikipedia.org/wiki/Object-relational_mapping>`_
+for PHP 5.4+ that provides transparent persistence for PHP objects. It uses the Data Mapper
```

Comment:
5.4+ isn't true anymore for some time already 😁

Reply:
AFAIK no, this PR targets 2.6 anyway. If required, it could be manually backported to 2.5 branch.

Reply:
I changed the version, but I'll have to add https for the wikipedia link.